collector/src/main.cpp

```diff
@@ -603,10 +607,14 @@ void CMonitorCollectorApp::parse_args(int argc, char** argv)
         printf("Option --remote-port=%lu provided but the --remote-ip option was not provided\n", m_cfg.m_nRemotePort);
         exit(53);
     }
+    if ((!m_cfg.m_strRemoteAddress.empty() && m_cfg.m_nRemotePort > 0) && m_cfg.m_strRemote.empty()) {
+        printf("Option --remote-ip and remort-port provided but the --remote option was not provided\n");
+        exit(54);
+    }
```

Comment:
let's also add validation about the contents of the m_cfg.m_strRemote variable: it's allowed to provide only "prometheus" or "influxdb" (just add a to-lowercase transformation before the check to be case insensitive)